chartfx-chart/src/main/java/de/gsi/chart/plugins/TableViewer.java

```diff
@@ -257,6 +251,7 @@ private ColumnType(final int dimIdx, final String label, final boolean errorCol,
      * @author akrimm
      */
     protected class DataSetsModel extends ObservableListBase<DataSetsRow> {
+        static final double DEFAULT_MAX_WIDTH = 5000.0F;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4989692)